pkg/network/controllers/vm_test.go

```diff
@@ -43,6 +43,10 @@ import (
 )
 
 var _ = Describe("VM Network Controller", func() {
+	const (
+		net1Name    = "net1"
+		net1NADName = "net1-nad"
```

Comment:
nit: I would hardcode the names, because I dont see the value in having them in this particular case.

Reply:
They will be used more in the follow-up.
No need to have repeated hard-coded string literals IMO.

Reply:
What I do not like about these names are that they repeat the value, which gives no added value.
What does "net1" represents? A secondary network? If so, name the const as such to give meaning.

To emphasize, these are not valid constant or variable names:
- one = 1
- num1 = 1
- foo = "foo"

Reply:
Renamed the constants to `secondaryNetName` and `nadName`.